docs/source/install.rst

```diff
@@ -5,7 +5,16 @@ Building and Installing
 Prerequisites
 =============
 
--  A ROCm enabled platform, more information `here <https://rocm.github.io/>`_.
+* If using the rocBLAS backend on an AMD machine:
+
+  * A ROCm enabled platform, more information `here <https://rocm.github.io/>`_.
+  * A compatible version of rocBLAS
+  * A compatible version of rocSOLVER for full functionality
+
+* If using the cuBLAS backend on a Nvidia machine:
+
+  * A HIP enabled platform, more information `here <https://rocmdocs.amd.com/en/latest/Installation_Guide/HIP-Installation.html#nvidia-platform/>`_.
```

Comment:
The slash that you often see at the end of a URL is a representation of the directory. For example, in the case of `https://rocm.github.io/`, that's the root directory.

This URL ends with a fragment-id (noted by the `#`). At that point, there shouldn't be any more slashes. The one on the end here actually breaks the reference to the `nvidia-platform` anchor (at least in Firefox).